docs/transparent-validation/rte/v1.1.0/specification.md

```diff
@@ -166,6 +166,48 @@ connector and can be enabled/disabled on demand:
 | 0 - low                 | disabled (by default)         |
 | 1 - high                | enabled                       |
 
+Example flashing procedure:
+
```

Comment:
In addition, the mentioned-above section should be started with introduction: in which cases it will be usable to change SPI voltage

Reply:
Done. FAQ informs about SPI voltage selection